src/common.h

```diff
@@ -0,0 +1,6 @@
+
+#pragma once
+
+#if defined(DEBUG) || !defined(NDEBUG)
```

Comment:
why the if is different from the src/debug.c

Reply:
true, should be similar.

Reply:
also note that `common.h` is being #include-ed into all source files in `RedisGraph/src` via `-include src/common.h`. This is good for making global definitions that should reach all compilation units.